<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWindow</class>
 <widget class="QMainWindow" name="CalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>672</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>672</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>672</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>370</width>
    <height>672</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CalculatorWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonClear">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ะก</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPlus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMinus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSubstract">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMultiply">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonToCube">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x^3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button10inNum">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10^x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonInverse">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEquals">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>580</y>
      <width>231</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPlusMinus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMS">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMR">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ResultDisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>40</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMMinus">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMPlus">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMClear">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MC</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
